changelogs/fragments/mysql_variables_allow_uppercase_identifiers.yml

```diff
@@ -0,0 +1,6 @@
+---
+bugfix:
+  - mysql_variables - Add uppercase character pattern to regex to allow GLOBAL
+    variables containing uppercase characters.
+    This recognises variable names used in Galera, e.g. wsrep_OSU_method,
+    which break the normal pattern of all lowercase characters.
```

Comment:
```suggestion
bugfix:
  - mysql_variables - add uppercase character pattern to regex to allow GLOBAL
    variables containing uppercase characters.
    This recognizes variable names used in Galera, for example, ``wsrep_OSU_method``,
    which breaks the normal pattern of all lowercase characters (https://github.com/ansible-collections/community.mysql/pull/501).
```

Reply:
sorry i made a mistake in the key

Reply:
bugfix -> bugfixes

Reply:
Please make any changes you think are suitable. I only started working with Ansible last week so I'm really happy you were even considering accepting the change.